packages/run-protocol/test/reserve/setup.js

```diff
@@ -59,6 +59,14 @@ const setupReserveBootstrap = async (
   };
 };
 
+/**
+ * @typedef {{
```

Comment:
When is this better than 
```
  * @typedef {Object} ReserveKit
  * @property {
```

Reply:
Style question. IMO we should use the TS syntax whenever we don't need to document with JSDoc. In this particular case I found the `import` calls to be hard to read when embedded in JSDoc.